src_assets/common/assets/web/config.html

```diff
@@ -517,6 +517,21 @@ <h1 class="my-4">Configuration</h1>
           <pre>   0: +HDMI-1 1920/518x1200/324+0+0 HDMI-1</pre>
         </div>
       </div>
+      <div class="mb-3">
+          <label for="qp" class="form-label">Preloaded Frame Count</label>
+          <input
+            type="number"
+            class="form-control"
```

Comment:
You should probably enforce a min (2) / max (12) here as well.

Reply:
That's the plan, main reason why I kept this as draft for the moment.

Reply:
Example in the port option.

```html
        <input
          type="number"
          min="0"
          max="65529"
          class="form-control"
          id="port"
          placeholder="47989"
          v-model="config.port"
        />
```